src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -582,6 +582,71 @@ void cannotCreateTlrRequestWhenUserHasLoanForSomeItemsOfInstance() {
       hasParameter("userId", usersFixture.jessica().getId().toString()))));
   }
 
+  @Test
+  void cannotCreateTlrRequestWhenUserHasAlreadyRequestedInstanceOrItemTlrOnRequestLevelTitle() {
+    reconfigureTlrFeature(TlrFeatureStatus.ENABLED);
+
+    IndividualResource instanceMultipleCopies = instancesFixture.basedUponDunkirk();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    UUID instanceId = instanceMultipleCopies.getId();
+
+    buildItem(instanceId, UUID.randomUUID().toString());
+    requestsClient.attemptCreate(makeRepresentation(pickupServicePointId, instanceId));
+
+    Response postResponse = requestsClient.attemptCreate(makeRepresentation(pickupServicePointId, instanceId));
+
+    assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
+    assertThat(postResponse.getJson(), hasErrorWith(allOf(
+      hasMessage("This requester already has an open request for this instance"),
+      hasParameter("requesterId", usersFixture.james().getId().toString()),
+      hasParameter("instanceId", instanceId.toString()))));
+  }
+
+  private JsonObject makeRepresentation(UUID pickupServicePointId, UUID instanceId) {
```

Comment:
and move these private methods downward please

Reply:
done